cache/diskstore-33bf9075af557542349b743962cd0f6b.json

```diff
@@ -0,0 +1 @@
+{"expireTime":1623433410152,"key":"30064fb775f8dea5b7b07dbd181b6f52","val":[1623433350136,1623433350151]}
```

Comment:
asking for my own clarity.  I haven't seen these files checked in before.  Are these new files or should they be checked in?

Reply:
Agreed! I suspect these come from github desktop and should probably be added to `.gitignore`

Reply:
Actually, I think this is a different issue altogether more related to the order in which NestJS loads modules - because the environment variables were added to `env.json`, which is parsed after the CacheModule is registered. This would be the same reason why the integration tests failed.

Reply:
Fixed in 7525f87